lib/improver/cli/update_grid_metadata.py

```diff
@@ -92,9 +56,6 @@ def process(cube):
             Processed Cube.
```

Comment:
This was never right - `process` was returning a boolean describing whether or not the cube had been modified in place.

Reply:
Yep. It was wrong, but now it's right 👍